Sources/XMTP/Messages/Invitation.swift

```diff
@@ -8,17 +8,15 @@
 import Foundation
 import XMTPProto
 
-typealias InvitationV1 = Xmtp_MessageContents_InvitationV1
+public typealias InvitationV1 = Xmtp_MessageContents_InvitationV1
 
 extension InvitationV1 {
-	@available(iOS 16.0, *)
 	static func createRandom(context: InvitationV1.Context? = nil) throws -> InvitationV1 {
 		let context = context ?? InvitationV1.Context()
 		let randomBytes = try Crypto.secureRandomBytes(count: 32)
-		let regex = #/=*$/#
 		let randomString = Data(randomBytes).base64EncodedString()
-			.replacing(regex, with: "")
-			.replacingOccurrences(of: "/", with: "-")
+			.replacingOccurrences(of: "=*$", with: "", options: .regularExpression)
```

Comment:
Changed this so we can stop relying on iOS 16 only regex stuff.

Reply:
Amazing.